rand-core/Cargo.toml

```diff
@@ -4,9 +4,9 @@ version = "0.1.0-pre.0"
 authors = ["The Rust Project Developers"]
 license = "MIT/Apache-2.0"
 readme = "README.md"
-repository = "https://github.com/rust-lang-nursery/rand"
+repository = "https://github.com/rust-lang-nursery/rand/tree/master/rand-core"
```

Comment:
I don't think this is right, as you can't do `git checkout https://github.com/rust-lang-nursery/rand/tree/master/rand-core`. This should just be the base repository, and cargo figures things out.